CDTDatastore/CDTReplicator/CDTReplicator.m

```diff
@@ -50,7 +50,33 @@ @interface CDTReplicator ()
 @property (nonatomic) BOOL started;
 
 @end
-
+/*
```

Comment:
You could say something like:

```
The CDTReplicator class is "fire and forget". This is so that an application 
can choose not to maintain a reference to an instance of CDTReplicator 
without the instance being immediately deallocated (and so failing to complete 
its replication).

This is implemented by each instance retaining itself by assigning to its 
own XXX property when replication is started. Care must be taken to ensure 
that this extra retain is released when the replication is complete.

This is implemented by retaining and releasing at the points noted in
the lifecycle diagram below:
```